modules/methods.js

```diff
@@ -0,0 +1,60 @@
+export default class Methods {
```

Comment:
- [ ] As this class is holding all of your book logic like(add, remove, store, and ...) I think it would be better to give it a descriptive name(class Books, class MyBooks, class BooksData...) as a javascript best practice.